src/applications/claims-status/reducers/serialize.js

```diff
@@ -0,0 +1,64 @@
+// NOTE: I think that in the long term it will make sense to move
+// this logic into the backend, but doing it here for now makes it
+// easier to modify this in the short term
+//
+// This does 2 things:
+// 1) Grabs all of the supporting documents associated with
+//    tracked items and embeds them within their respective
+//    tracked items in a new `documents` key that has been
+//    added to each tracked item
+// 2) Gets a new array of supporting documents that only
+//    includes the documents that are not associated with
+//    any tracked items. These items should already be
+//    embedded in their respective tracked items anyways,
+//    so they are no longer needed
+const isLighthouseClaim = claim => claim.type === 'claim';
+
+const isAssociatedWithAnyTrackedItem = doc => doc.trackedItemId !== null;
+
+const isAssociatedWithTrackedItem = item => doc =>
+  doc.trackedItemId === item.id;
```

Comment:
Would you ever run into a situation where `doc.trackedItemId` AND `item.id` are both undefined?

Reply:
I don't think so. It's expected that the supporting document can have a `trackeditemId` of `null`, but if any of the tracked items have an `id` of `null` then there is a bigger issue. I could potentially do a precheck to see if the `id` field on the tracked item is `null` or `undefined` so it doesn't make it to this part if you think it's worth it

Reply:
I was asking out of curiosity. Go ahead and go with what you have.